RUFAS/output_manager.py

```diff
@@ -144,6 +165,8 @@ def add_log(self, name: str, msg: str, info_map: Dict[str, Any]) -> None:
         info_map["timestamp"] = self._get_timestamp(include_millis=True)
         key = self._generate_key(name, info_map)
         self._add_to_pool(self.logs_pool, key, msg, info_map)
+        if self.__log_type == LogType.LOGS:
+            sys.stdout.write(f"{self.__metadata_prefix}[LOG]: {name}: {msg}\n")
```

Comment:
Applies to other instances as well.
```suggestion
            sys.stdout.write(f"[Time Stamp][LOG][{self.__metadata_prefix}] {name}: {msg}\n")
```

Reply:
Addressed with logs, errors, and warnings.